src/Framework/Framework/Hosting/DotvvmRequestContext.cs

```diff
@@ -75,7 +75,7 @@ public bool IsPostBack
         public ResourceManager ResourceManager => _resourceManager ?? (_resourceManager = Services.GetRequiredService<ResourceManager>());
 
         /// <summary>
-        /// Gets the <see cref="ModelState"/> object that manages validation errors for the viewmodel.
+        /// Gets the <see cref="ModelState"/> object that manages validation errors for the and static command arguments.
```

Comment:
Revert this, and explicitly mention that static commands are using a different class.